lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -50,6 +51,19 @@ export class RefreshTokenClient extends BaseClient {
         );
 
         responseHandler.validateTokenResponse(response.body);
+        if (response.body?.refresh_token) {
+            atsMeasurement?.endMeasurement({
+                success: true,
+                refreshTokenSize: response?.body?.refresh_token.length
+            });
+        }
+        else {
+            atsMeasurement?.endMeasurement({
+                success: true,
+                refreshTokenSize: -1
```

Comment:
What are the pros/cons of using -1 vs simply not providing a value in this case?

Reply:
Would rather this be undefined than -1, as -1 would throw off any calculations.

Reply:
If you want to use a number, it should be 0.

Reply:
This should translate to null/empty in Aria if there is no token.  It would be helpful when looking at the telemetry to infer that a null in the RT size implies no token was returned, where a 0 would indicate a token was returned but it was zero bytes long (a code error somewhere on the client or server)

Reply:
Using 0 in case of RT length not available (in event of token not returned in response body), and undefined when there is an error in the token acquisition

Reply:
Please leave this undefined when no RT is returned.  Otherwise, we won't be able to differentiate not getting a token back and getting back a token that is zero bytes in size.

Reply:
There is some issue, the code is not updated . Can you please check https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5063/files -> Changes from all commits , in case it still shows the older code, I'll need to check the commit sequence on my end, thanks

Reply:
> Please leave this undefined when no RT is returned. Otherwise, we won't be able to differentiate not getting a token back and getting back a token that is zero bytes in size.

Is this something that will actually happen? Seems like were optimizing for a scenario that won't happen in practice (and in my opinion, doesn't need to be treated differently).

Reply:
We should handle missing data consistently.  If we don't have something from the server or the user we shouldn't log a default there.